ecosystem/sep-0006.md

```diff
@@ -367,18 +368,50 @@ Name | Type | Description
 `fee_percent` | float | (optional) Percentage fee (if any). In units of percentage points.
 `extra_info` | object | (optional) JSON object with additional information about the deposit process.
 
+`instructions` fields:
+
+An object with SEP-9 financial account fields as keys and its values are objects with the following fields:
+
+Name | Type | Description
+-----|------|------------
+`description` | string | A human-readable description of the field. This can be used to provide any additional information about fields that are not defined in the SEP-9 standard.
+
 `extra_info` fields:
 
 Name | Type | Description
 -----|------|------------
 `message` | string | (optional) Additional details about the deposit process.
 
+##### Examples
+
+Bank payment example:
+```json
+{
+  "id": "9421871e-0623-4356-b7b5-5996da122f3e",
+  "instructions": {
+    "organization.bank_number": {
+      "value": "121122676",
+      "description": "US bank routing number"
+    },
+    "organization.bank_account_number": {
+      "value": "13719713158835300",
+      "description": "US bank account number"
+    }
+  }
+}
+```
```

Comment:
We're not removing `how`, and its still required, so we need to have it present in our examples.